FWCore/Services/test/test_resourceInformationService_cfg.py

```diff
@@ -0,0 +1,30 @@
+# Test ResourceInformationService
+# The shell script will examine its printout
+import FWCore.ParameterSet.Config as cms
+
+# This class is a hack just for the test to get
+# 'gpu-foo' into @selected_accelerators
+class ProcessAcceleratorTest(cms.ProcessAccelerator):
+    def __init__(self):
+        super(ProcessAcceleratorTest,self).__init__()
+        self._labels = ["test1", "gpu-foo", "test2"]
+        self._enabled = ["test1", "gpu-foo", "test2"]
+    def labels(self):
+        return self._labels
+    def enabledLabels(self):
+        return self._enabled
```

Comment:
If the intention is to have all accelerator labels enabled, this could be just
```suggestion
        return self._labels
```
and `self._enabled` removed.

Reply:
That is better. Done. Thanks.